stl/inc/xutility

```diff
@@ -4499,7 +4499,9 @@ template <class _InIt, class _Sent, class _OutIt>
 _CONSTEXPR20 _OutIt _Copy_unchecked(_InIt _First, _Sent _Last, _OutIt _Dest) {
     // copy [_First, _Last) to [_Dest, ...)
     // note: _Copy_unchecked has callers other than the copy family
-    if constexpr (_Sent_copy_cat<_InIt, _Sent, _OutIt>::_Bitcopy_assignable) {
+    if constexpr (_Is_vb_iterator<_InIt> && _Is_vb_iterator<_OutIt>) {
+        return _Copy_vbool(_First, _Last, _Dest);
+    } else if constexpr (_Sent_copy_cat<_InIt, _Sent, _OutIt>::_Bitcopy_assignable) {

```

Comment:
This is causing an unreachable code warning in the modules tests (there may be other occurrences). The issue is that this says `if constexpr (...) { return ...; } else if constexpr (...) { stuff; } stuff; return ...;`. If this were restructured to be: `if constexpr (...) { return ...; } else { ALL OLD CODE }` then the compiler wouldn't see multiple returns and wouldn't warn. Although the warning is obnoxious, the pattern to avoid it will result in less debug codegen, so it's worthwhile.